src/Sample.xoo

```diff
@@ -10,3 +10,4 @@ public class Sample1 {
     System.out.println("hello" + " xoo");
   }
 }
+// xoo
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:40071/organizations/org3/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:40071/project/issues?id=sample&pullRequest=6885&issues=AWlTtGsaPTLNrAwP2TxW&open=AWlTtGsaPTLNrAwP2TxW)

[ISSUE_KEY]: AWlTtGsaPTLNrAwP2TxW